app/portainer/views/settings/authentication/settingsAuthentication.html

```diff
@@ -94,33 +94,49 @@
               </div>
             </div>
 
+            <!-- Anonymous mode-->
             <div class="form-group">
-              <label for="ldap_username" class="col-sm-3 col-lg-2 control-label text-left">
-                Reader DN
-                <portainer-tooltip position="bottom" message="Account that will be used to search for users."></portainer-tooltip>
-              </label>
-              <div class="col-sm-9 col-lg-10">
-                <input type="text" class="form-control" id="ldap_username" ng-model="LDAPSettings.ReaderDN" placeholder="cn=readonly-account,dc=ldap,dc=domain,dc=tld">
+              <div class="col-sm-12">
+                <label for="anonymous" class="control-label text-left">
+                  Anonymous mode
+                  <portainer-tooltip position="bottom" message="Only enable this option if the server is configured for Anonymous access"></portainer-tooltip>
+                </label>
+                <label class="switch" style="margin-left: 20px;">
+                  <input type="checkbox" id="anonymous" ng-model="formValues.Anonymous"><i></i>
+                </label>
               </div>
             </div>
+            <!-- !Anonymous mode-->
 
-            <div class="form-group">
-              <label for="ldap_password" class="col-sm-3 col-lg-2 control-label text-left">
-                Password
-              </label>
-              <div class="col-sm-9 col-lg-10">
-                <input type="password" class="form-control" id="ldap_password" ng-model="LDAPSettings.Password" placeholder="password">
+            <div ng-if="!formValues.Anonymous">
+              <div class="form-group">
+                <label for="ldap_username" class="col-sm-3 col-lg-2 control-label text-left">
+                  Reader DN
+                  <portainer-tooltip position="bottom" message="Account that will be used to search for users."></portainer-tooltip>
+                </label>
+                <div class="col-sm-9 col-lg-10">
+                  <input type="text" class="form-control" id="ldap_username" ng-model="LDAPSettings.ReaderDN" placeholder="cn=readonly-account,dc=ldap,dc=domain,dc=tld">
+                </div>
               </div>
-            </div>
 
+              <div class="form-group">
+                <label for="ldap_password" class="col-sm-3 col-lg-2 control-label text-left">
+                  Password
```

Comment:
Maybe add a tooltip to state that leaving this field empty keeps the current password unchanged?